src/utils/subgraph.ts

```diff
@@ -16,6 +16,10 @@ import {
   AssetsFreePrice_dispensers as AssetFreePriceDispenser
 } from '../@types/apollo/AssetsFreePrice'
 import { AssetPreviousOrder } from '../@types/apollo/AssetPreviousOrder'
+import {
+  HighestLiquidiyAssets_pools as HighestLiquidiyAssetsPools,
```

Comment:
not sure where exactly this is coming from but throughout code small typo with a missing `t` like `HighestLiquidiyAssetsPools` which should be `HighestLiquidityAssetsPools `

Reply:
right missed that 😃 , it comes from the typing generated by the highest liquidity query, I'll update that also